tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -718,9 +724,9 @@ class MklFusedBatchNormOp : public OpKernel {
         std::memcpy(batch_variance_data, variance_data, depth_ * sizeof(U));
       }
     } catch (mkldnn::error& e) {
-      string error_msg = "Status: " + std::to_string(e.status) +
-                         ", message: " + string(e.message) + ", in file " +
-                         string(__FILE__) + ":" + std::to_string(__LINE__);
+      string error_msg = "Status: " + std::to_string(e.status) + ", message: " +
+                         string(e.message) + ", in file " + string(__FILE__) +
+                         ":" + std::to_string(__LINE__);
```

Comment:
This reformatting shouldn't happen. Please make sure you're using `clang-format` 8.0.1 and reformat. (Or manually revert this for now.)